src/components/challengeWidget/challengeWidget.css

```diff
@@ -1,31 +1,31 @@
 .challengeWidget {
-  width: 700px;
-  height: 250px;
-  border: 15px solid burlywood;
+  width: 100%;
+  border: 1px solid burlywood;
   background-color: white;
 }
 
-h1 {
+.challengeWidget h2 {
   color: black;
-  font-family: sans-serif;
+  font-family: inherit;
 }
-p {
-  font-family: Verdana, Geneva, Tahoma, sans-serif;
+.discription {
```

Comment:
You will have to prefix every rule in this file with `.challengeWidget` so the styles will not affect other components.